examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/controlexample/ExpandBarTab.java

```diff
@@ -80,6 +80,7 @@ void createExampleWidgets () {
 		item.setHeight(composite.computeSize(SWT.DEFAULT, SWT.DEFAULT).y);
 		item.setControl(composite);
 		item.setImage(instance.images[ControlExample.ciClosedFolder]);
+		item.setExpanded(true);
```

Comment:
Why has this been added and removed from at some other place? Seems like an erroneous revert to me.